### PR TITLE
enhancement/admin

### PR DESCRIPTION
**Pull request by alexfradiani**
This PR:

- Minor adjustments in the admin setup to remove branding links from the agency that created AdminBro.
- Basic layout to ramp up a dashboard.
- Launch-file to enable live debugging from vscode.

### CODE REVIEW

File: .adminbro/.entry.js [hunk 1]
```diff
@@ -1 +1,3 @@
-AdminBro.UserComponents = {};
```
Comment (alexfradiani) [inline .adminbro/.entry.js]: I still think this file is not necessary, and its output is a bit dirty. The adminbro's dev recommended to leave it out of the repo.

File: src/admin/dashboard.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Box, Header } from 'admin-bro';
+
+const Dashboard = () => {
+  return (
+    <Box variant="grey">
```
Comment (alexfradiani) [inline src/admin/dashboard.jsx:6]: _Box_ is a component built-in with the admin, but this is essentially a React component..., so we should be able to do any desired customization in a real project.

File: src/admin/decs.d.ts [hunk 3]
```diff
@@ -0,0 +1,2 @@
+// express plugin for adminbro doesn't have types yet. This is a workaround.
+declare module '@admin-bro/express';
```
Comment (alexfradiani) [inline src/admin/decs.d.ts:2]: only workaround for the moment, they don't have types in the module.

File: src/server.ts [hunk 4]
```diff
@@ -22,7 +21,11 @@ const handleConnection = async (connection: Connection) => {
     ],
     rootPath: '/admin',
     branding: {
-      companyName: 'Rootstrap'
+      companyName: 'Rootstrap',
```
Comment (pablanco) [inline src/server.ts:24]: For the next, we can maybe use a `const` file for this params.

**Review by pablanco — APPROVED** [review]

**Review by Alxzu — CHANGES_REQUESTED** [review]
Can you make sure the merge against `master` is complete? I'm seeing some errors locally due to `User.age`

File: src/admin/decs.d.ts (hunk 3 above)
Comment (Alxzu) [reply src/admin/decs.d.ts:2]: Maybe we can use a `// TODO:` or `// FIX: ` to start these comments above in order to have a way of looking tech debt inside the code later ? wdyt?

File: .adminbro/.entry.js [hunk 5]
```diff
@@ -1 +1,3 @@
...
+AdminBro.UserComponents = {}
+import Component1 from '../src/admin/dashboard'
+AdminBro.UserComponents.Component1 = Component1
```
Comment (Alxzu) [inline .adminbro/.entry.js]: Can we have another name for this?

**Review by Alxzu — APPROVED** [review]

**Pull request merged**